app/index.js

```diff
@@ -258,16 +258,47 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   installDeps: function () {
-    if (!this.options['skip-install']) {
-      this.on('end', function () {
-        this.options.plugins.push('grunt-karma');
-        if (this.options.coffee) {
-          this.options.plugins.push('coffee-script');
+
+    function save (where, things, cb) {
+      var sortedObject = require("sorted-object");
+      var writeFileAtomic = require("write-file-atomic");
+      var saveTarget = path.resolve(where, "package.json");
+
+      fs.readFile(saveTarget, function (er, data) {
```

Comment:
This is not the way to handle files with Yeoman. See http://yeoman.io/authoring/file-system.html